src/from_dom.js

```diff
@@ -798,8 +797,12 @@ function markMayApply(markType, nodeType) {
   }
 }
 
-function findSameTypeInSet(mark, set) {
+function sameMark(mark, other) {
```

Comment:
This already exists as the mark's `eq` method.

Reply:
OK,this is my negligence and has been modified to use `eq`.